src/main/webapp/style.css

```diff
@@ -0,0 +1,236 @@
+body {
+  background-color: #ececec;
+  color: black;
+  font-family: Aksara Bali Galang;
+}
+
+main {
+  margin: auto;
+  max-width: 1035px;
+  width: 100%;  
+}
+
+#content {
+  margin: auto;
+  max-width: 100%;
+}
+
+h1 {
+  color: white;
+  font-family: Ruluko;
+  font-size: 40px;
+  font-weight: normal;
+  line-height: 69px;
+  margin: 10px 20px;
+}
+
+h2,
+h3 {
+  color: #856c8b;  
+  text-align: center;
+}
+
+h5 {
+  color: #787878;  
+  display: block;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: normal;
+  height: 26px;
+  line-height: 49px;
+  margin: auto;
+  margin-bottom: 15px;
+  width: 955px;
+}
+
+p { 
+  color: black;  
+  display: block;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: normal;
+  line-height: 25px;
+  margin: auto;
+  width: 960px;
+}
+
+a {
+  color: #856c8b;  
+  font-size: 20px;
+  font-style: normal;
+  font-weight: normal;
+  height: 39px;
+  line-height: 43px;
+  text-decoration: none;
+}
+
+input {
+  display: block;  
+  height: 25px;
+  margin: 8px auto;
+  max-width: 1035px;
+  width: 100%;
+}
+
+input:hover {
+  opacity: 0.7;
+}
+
+textarea {
+  box-shadow: 0 4px 4px #00000040;
+  display: block;  
+  height: 70px;
+  margin: auto;
+  max-width: 1035px;
+  width: 100%;
+}
+
+textarea:hover {
+  opacity: 0.7;
+}
+
+button {
+  background-color: #856c8b;
+  border: none;
+  color: white;
+  cursor: pointer;
+  display: block;
+  font-size: 15px;
+  margin: 15px auto 25px;
+  padding: 10px 20px;
+  text-align: center;
+}
+
+header {
+  background: #856c8b;
+  border: 1px solid black;
+  box-shadow: 0 4px 4px #00000040;
+  box-sizing: border-box;
+}
+
+#school-search {
+  float: right; 
+  height: 35px; 
+  position: absolute;
+  right: 45px;
+  top: 35px;
+  width: 300px;
+}
+
+search-button {
+  cursor: pointer;
+  position: absolute;
+  right: 57px;
+  top: 55px;  
+}
+
+#comments-container {
+  background: white;
+  border: 0.25px solid black;  
+  box-shadow: 0 4px 4px #00000040;
+  height: 379px;
+  margin: auto;
+  max-width: 1035px;
+  width: 100%;
+}
+
+#data-holder {
+  font-size: 25px;
+  margin-bottom: 50px;
+  margin-top: 40px;
+}
+
+.school-header {
+  background: white;
+  box-shadow: 0 4px 4px #00000040;  
+  margin: auto;
+  margin-top: 20px;
+}
+
+#school-name {
+  color: #856c8b; 
+  font-size: 30px; 
+  margin: auto;
+  padding: 22px;
+  text-align: center;
+}
+
+#school-desc {
```

Comment:
I'm noticing that these sections have a lot of repeated styles. I recommend grouping all of the shared styles together in one rule and specify all of the ID selectors they apply to. Then separately break out each one like you've done, but only add the unique styles.